test/Feature/UpsertSpec.hs

```diff
@@ -43,6 +45,20 @@ spec =
             , matchHeaders = ["Preference-Applied" <:> "resolution=merge-duplicates", matchContentTypeJson]
             }
 
+        when (actualPgVersion >= pgVersion110) $
+          it "INSERTs and UPDATEs rows on composite pk conflict for partitioned tables" $
+            request methodPost "/partitioned_a" [("Prefer", "return=representation"), ("Prefer", "resolution=merge-duplicates")]
+              [json| [
+                { "id": 4, "name": "second", "id_ref": 2},
+                { "id": 6, "name": "first", "id_ref": 1 }
+              ]|] `shouldRespondWith` [json| [
+                { "id": 4, "name": "second", "id_ref": 2 },
+                { "id": 6, "name": "first", "id_ref": 1 }
```

Comment:
I see the `id: 6` value is inserted here, but is the `id: 4` updated in any column? I see it already has "second" in the fixtures.

Reply:
Yes, it's a composite primary key `(id, name)`. The column that is updated is `id_ref`, from `null` to `2`.

Reply:
I see, then it's good to leave it as it is :+1: